targets/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+matrix:
+  fast_finish: true
+env:
+  global:
+  - secure: Qn2ljQmSdv/A8a8SgDwcoJJQrlSPIBQ0gIG2JAjRx0mUA7Xwmvb5AYHvgMRDTHwXAFlGHtPlwoPgWa6mKo1zlu1lROiycuhVOYhIcD7SY9XCJ7d4fVSg03KRaRRmDEb9OVIyioNW7Bq1KMtPJVG0+5b+GYXWVm+gZrc7A9HP9ADWGpQuDvpU1UMAp2SQlL1ZFOr7HzF8hcjJkE7IfhuUzI4l1cYJio/X0baBvJ4KEI3P6fjlzYtmQwWNqhz+8kzuEbMdqSbWIG5CADxVonp9OFlCRSwL2lusdF9FMgO31+4H17JBz96CW+Tpnvlyisefj87zT/OK05xE1RmtSz55cf4bOfa4my8dlptqIZIEQBFq51Zcjsf/pQ8RVIi6tZXnsyjdJH5lq0Vt8Rzu7KWJhR7fMc1kV8orm9ngLcSBMRJR7uxvvH4X5+5FMzNFCQPMLUGptjkfR5LXN+KCVovL/eEaXGFS/0N4TDSMI1kETgT9HexKqxLJR0bEUFbMw1dopzKqLBiGjQ/KcdfCQNY5FQBcu4nvaoiplE5054kE5Ol+cLuRusn5QJg0IQFoZd/YsRKjeA1cBTDafMPKn6SLENj3xfNkUN6rq95saJUhdQDYpzC92n1X3y4FmUfr1bmhkpJuR4GPxeZNrCzWYjAKTPRzl+rLfbp9mXKTPnRrRM0=
+script:
+- sh publish_registry.sh
```

Comment:
I don't see any `COZY_BUILD_BRANCH` variable here for the publish script, it should break without it.

Reply:
Right, I forgot to move it from https://github.com/cozy/cozy-drive/pull/465/files#diff-354f30a63fb0907d4ad57269548329e3R11

Reply:
I moved it to the correct `.travis.yml` file: https://github.com/cozy/cozy-drive/pull/465/files#diff-24b4b31e00acf4e2760f0c8dc2e94493R6
with value: `latest-drive` or `latest-photos`